maintenance.md

```diff
@@ -164,14 +164,39 @@ In Regular update, do the following as part of the update of each CRD-providing
 
 ![CSA Update](./csa_update.svg)
 
-1. Check the [release page](https://docs.ceph.com/en/latest/releases/).
-2. Check the [build ceph](https://docs.ceph.com/en/latest/install/build-ceph/) document and [README.md](https://github.com/ceph/ceph/blob/main/README.md).
-   1. If other instructions are needed for `ceph/build.sh`, add the instructions.
+***NOTE:*** The rook image is based on the ceph image. So upgrade the [rook](#rook) image next.
+
+***NOTE:*** (Optional) You can delete the old version's directory and the related CI jobs. For example, if you consider vP.Q.R will not be used anymore, delete the `ceph/vP-Q-R` directory and the following CI jobs.
+- Github Actions: `build_cephP-Q-R` job.
+- CircleCI: `build-cephP-Q-R` job and `build-cephP-Q-R-container` job.
+
+### Directory naming convention
+
+- `ceph/latest`: The latest ceph build settings are located.
+- `ceph/vX-Y-Z`: Build settings for the specific version of Ceph are located.
+
+### When you update based on the `ceph` directory
+
+1. Check the version in the `ceph/latest/TAG` file. If the version is vX.Y.Z.W, then copy the `ceph/latest` directory to the `ceph/vX-Y-Z` directory and add the following CI jobs.
+   - Github Actions: `build_cephX-Y-Z` job.
+   - CircleCI: `build-cephX-Y-Z` job and `build-cephX-Y-Z-container` job.
+2. Check the [release page](https://docs.ceph.com/en/latest/releases/).
+3. Check the [build ceph](https://docs.ceph.com/en/latest/install/build-ceph/) document and [README.md](https://github.com/ceph/ceph/blob/main/README.md).
+   1. If other instructions are needed for `ceph/latest/build.sh`, add the instructions.
    2. If there are ceph runtime packages or required tool changes, update Dockerfile.
-3. Update the `version` argument on the `build-ceph` job in the CircleCI `main` workflow and the `build_ceph` job in the Github Actions `main` workflow.
-4. Update `BRANCH` and `TAG` files.
+4. Update the `version` argument on the `build-ceph-latest` job in the CircleCI `main` workflow and the `build_ceph` job in the Github Actions `main` workflow.
+5. Update `BRANCH` and `TAG` files in the `ceph/latest` directory.
 
-***NOTE:*** The rook image is based on the ceph image. So upgrade the [rook](#rook) image next.
+### When you update based on the `cephX-Y-Z` directory
+
+1. If the **new** version is vA.B.C.D, then copy the `ceph/vX-Y-Z` directory to the `ceph/vA-B-C` directory and add the following CI jobs.
```

Comment:
vX-Y-Z is the upstream version. If we need to change files on the directory, it is our artifacts (Dockerfile, build scripts, or patches). I think we don't need to copy this because the upstream version has not changed.
Could you explain your thoughts with specific version numbers?

Reply:
Got it.
I prefer waiting for a new v18 version, but it would be better to discuss this with the whole team.